### PR TITLE
chore(timeline server components) Moved timeline to server components

### PR DESCRIPTION
**Pull request by edge33**
### Description

Home page timeline components have been refactored to work as server components

## Type

- [ ] **Dependency upgrade**
- [ ] **Bug fix**
- [ ] **New feature**
- [x] **Dev change**

### CODE REVIEW

**Review by GiuliaTeggi — APPROVED** [review]

File: apps/web/Dockerfile [hunk 1]
```diff
@@ -23,12 +22,23 @@ COPY ./packages/building-blocks-sdk/ /app/packages/building-blocks-sdk/
...
 RUN npm run build --workspace=apps/web
 
+from base-deps as runner
+
+WORKDIR /app
+
+ARG NEXT_PUBLIC_API_ENDPOINT
+ENV LOG_LEVEL=trace
+ENV NEXT_PUBLIC_API_ENDPOINT=$NEXT_PUBLIC_API_ENDPOINT
+ENV NODE_ENV=production
```
Comment (williamwd) [inline apps/web/Dockerfile:37]: This is also provided at runtime

File: apps/web/app/[locale]/[event]/components/EventTypeSelector.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+import { useTranslations } from "next-intl";
+import { usePathname, useRouter } from "next/navigation";
+
+/**
+ * this is the only client component for search,
+ * we don't know yet if we are going to need it to work with no js,
+ * so it would be cumbersome to have both a js/non js solution as of now
+ * we can change it in the future if ever need to
+ */
```
Comment (GiuliaTeggi) [inline apps/web/app/[locale]/[event]/components/EventTypeSelector.tsx:11]: I think we can indeed say that the app needs to work with no js, as the client has said several times, and that should be the priority. However I'll approve this for now, given the design for this component needs to be reviewed.

**Review by GiuliaTeggi — APPROVED** [review]

File: apps/web/app/[locale]/page.tsx [hunk 3]
```diff
@@ -8,6 +8,7 @@ type Props = {
 };
 
 export default (props: Props) => {
+  console.log("PUBLIC_ENDPOINT", process.env.NEXT_PUBLIC_API_ENDPOINT);
```
Comment (GiuliaTeggi) [inline apps/web/app/[locale]/page.tsx:11]:
```suggestion
```

File: apps/web/app/[locale]/page.tsx [hunk 4]
```diff
@@ -8,6 +8,7 @@ type Props = {
...
   const path = `${props.params.locale}/${routes.events.slug}`;
   redirect(path, RedirectType.replace);
```
Comment (GiuliaTeggi) [inline apps/web/app/[locale]/page.tsx:13]: Could you double check that this path without the start `/` works fine?

**Pull request merged**